main.py

```diff
@@ -156,6 +180,40 @@ def event_retrieve(self, event):
 
         self._execute_callback(event, box, error)
 
+    @listen_to('kytos.storehouse.update')
+    def event_update(self, event):
+        """Update a box_id from namespace.
+
+        This method receive an event with the follow parameters:
+
+        namespace: namespace where the box is stored
+        box_id: the box identify
+        method: 'PUT' or 'PATCH', the default update method is 'PATCH'
+        data: a python dict with the data
+        """
+        error = False
+
+        try:
+            backend = FileSystem()
+            box = backend.retrieve(event.content['namespace'],
+                                   event.content['box_id'])
+        except (AttributeError, KeyError, TypeError, ValueError):
```

Comment:
This looks like too many errors being silenced. Are all of those really necessary?